Code/Core/Mem/MemDebug.cpp

```diff
@@ -9,6 +9,10 @@
 
 #if defined( MEM_DEBUG_ENABLED )
 
+#if defined( __GNUC__ ) && __GNUC__ >= 7
+    #pragma GCC diagnostic ignored "-Wimplicit-fallthrough"
+#endif
+
```

Comment:
I think we might be better off just changing the one switch statement the flags this so that we don't need to turn it off, potentially covering up other instances of this warning in this cpp and other cpp files in the library when it is built in Unity.

Reply:
Corrected. I'll upload next iteration soon